src/scrapers/visa-cal.ts

```diff
@@ -85,6 +84,7 @@ interface InitResponse {
     }[];
   };
 }
+// eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
 type CurrencySymbol = '₪' | string;
```

Comment:
I see the warning is because the type is finally identified as `string`, so we can remove the `'₪'`.

```suggestion
type CurrencySymbol = string;
```